omero/sysadmins/config.rst

```diff
@@ -2068,7 +2106,7 @@ omero.version
 ^^^^^^^^^^^^^
 Value dynamically set during the build
 
-Default: `5.6.3`
+Default: `5.6.1-SNAPSHOT`
```

Comment:
Same as above. Unexpected SNAPSHOT version

Reply:
see https://github.com/ome/omero-documentation/pull/2202#issuecomment-990805962

Reply:
This is coming from omero.server.

Reply:
I dig a bit of digging. As mentioned previously, ``omero.version`` is set dynamically during the build of omero-server cf. https://github.com/ome/omero-server/blob/v5.6.1/src/main/resources/omero-server.properties#L493
openmicroscopy depends on ``omero-blitz=5.5.8`` https://github.com/ome/openmicroscopy/blob/develop/etc/omero.properties#L211 which in term depends on ``omero-server=5.6.1``
Checking the content of the ``omero-server`` jar, ``omero.version`` value is set to ``5.6.1-SNAPSHOT``. This is an error

The value ``5.6.3`` will have been added manually to the doc and does not correspond to any value dynamically set into the properties file. The value set in the release version is ``omero.version=5.6.3-ice36-b228``.

The way to get that snapshot away is to have a new release of ``omero-server`` and ``omero-blitz`` and bump the version in ``openmicroscopy`` properties file

Reply:
This will require https://github.com/ome/omero-py/pull/312 to be merged and a new version of omero-py

Reply:
sorted